CONTRIBUTING.md

```diff
@@ -92,3 +93,83 @@ The following pages describe the Git development workflows for some other projec
 may also find some helpful advice from these projects:
 * [Matplotlib](https://matplotlib.org/devdocs/devel/development_workflow.html)
 * [SciPy](https://docs.scipy.org/doc/scipy/dev/contributor/development_workflow.html)
+
+## Restrictions on Generative AI Usage
+
+We expect authentic engagement in our community.
+
+- Do not post output from Large Language Models or similar generative AI as comments on
+  GitHub or our User Group, as such comments tend to be formulaic and low content.
+- If you use generative AI tools as an aid in developing code or documentation changes,
+  you are required to disclose the use of AI and ensure that you fully understand the
+  proposed changes and can explain why they are the correct approach.
+
+Make sure you have added value based on your personal competency to your contributions.
+Just taking some input, feeding it to an AI and posting the result is not of value to
+the project. To preserve precious core developer capacity, we reserve the right to
+rigorously reject seemingly AI generated low-value contributions.
+(_adopted from
```

Comment:
Looks like there's some misplaced `_` here.

Reply:
There's a second `_` later on - that is all italicized.